### PR TITLE
INT - update usage variable descriptions

### PR DESCRIPTION
**Pull request by jcollins-axway**
## Describe the changes

- Update how the various usage variables are described
- Remove offline usage variables on agents that will not use them

## Deploy preview link

- http://pr-434.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_api_manager/agent-variables/
- http://pr-434.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-1/
- http://pr-434.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_kafka_cluster/agent-variables/
- http://pr-434.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_azure_gateway/agent-variables/

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | df9817c |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-434.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by jcollins-axway** [comment]
@sbolosan and @vivekschauhan could you take a quick peek at the updates around usage variables?

**Review by robertengaxway — APPROVED** [review]
Changes I viewed all look good.
There was feedback that we did not document the "TRACEABILITY_SAMPLING_REPORTALLERRORS" variable, if we could add the default value and a description, that would be good

**Comment by jcollins-axway** [comment]
@robertengaxway I increased documentation on that variable, could you review again?

**Review by robertengaxway — APPROVED** [review]
I approve and thank you for adding the default values and minimum/maximum values

File: content/en/docs/connect_manage_environ/connected_agent_common_reference/trace_sampling.md [hunk 1]
```diff
@@ -60,6 +60,18 @@ TRACEABILITY_SAMPLING_PER_API=false
...
+
+The following is a sample value that is added to `ta_env_vars.env`, which will send 10% of the gateway transactions to Amplify Analytics. Including all errors.
+
+```shell
+TRACEABILITY_SAMPLING_PERCENTAGE=10
+TRACEABILITY_SAMPLING_PER_API=false
+TRACEABILITY_SAMPLING_REPORTALLERRORS=true
+```
+
+If the agent receives 100 transactions, evenly between 2 APIs, the first 10 will be sent to Amplify Analytics. If of the 90 that are not sent sampled any error codes exist then they too will be sent to Amplify Analytics
```
Comment (lbadenhop) [inline content/en/docs/connect_manage_environ/connected_agent_common_reference/trace_sampling.md:73]:
```suggestion
If the agent receives 100 transactions, evenly between 2 APIs, the first 10 (five from each API) will be sent to Amplify Analytics. If error codes exist in the remaining non-sampled percentage, then those transactions will also be sent.
```

**Pull request merged**